Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts

```diff
@@ -15,6 +16,11 @@ const formatDate = (
     ? moment(date).format(formatString)
```

Comment:
Use `preciseFormatDate` here as it is the same.

Reply:
Done.